plotly/tools.py

```diff
@@ -191,30 +198,46 @@ def set_config_file(plotly_domain=None,
     settings = get_config_file()
     if isinstance(plotly_domain, six.string_types):
         settings['plotly_domain'] = plotly_domain
+    elif plotly_domain is not None:
+        raise TypeError('Input should be a string')
```

Comment:
yeah, the way we say `domain` but mean `schema + domain` is a little confusing anyhow. we could definitely be smarter about it.